src/test/java/io/cdap/plugin/ToUTF8ConfigTest.java

```diff
@@ -55,9 +59,10 @@ public void testSingleFile() throws Exception {
     ToUTF8Action.ToUTF8Config config = new ToUTF8Action.ToUTF8Config(iso88591File.getFile(),
                                                                      destFolder.getPath() + "/" + UTF_8_FILE_NAME,
                                                                      null, "ISO-8859-1", false);
-    MockPipelineConfigurer configurer = new MockPipelineConfigurer(null);
+    MockCustomPipelineConfigurer configurer = new MockCustomPipelineConfigurer(null);
```

Comment:
I think we should ask about replacing this tests with tests for Config class. It will allow to avoid custom mocks for context instances.